client/gutenberg/components/jetpack-branding/index.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { InspectorControls } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import JetpackLogo from 'components/jetpack-logo';
+import './style.scss';
+
+const JetpackBranding = () => (
+	<InspectorControls>
+		<div className="jetpack-branding">
```

Comment:
Can we set the `className` directly on `InspectorControls`?

Reply:
No, we can't - which is why I did it that way.